### ISSUE TITLE
NonConverge

### ISSUE THREAD

**Issue by tyut11103** [cogsys-tuebingen/PAL#1]
Hello Mr maximus.mutschler:
    I have read your paper carefully and very interest in your work. The PAL agorithm you provided can automatically find a good learning rate without  the need of hand-designed step size schedules.
    I want reproduce the experiment of Resnet50 on ImageNet , BUT the training loss did not converge. The optimizer used the PalOptimizer come from your pytorch code ,and tried the recommended hyperparameters.  Can I get your hyperparameter configuration? 
    Waiting for your reply.  Best Regards
